src/adapters/suggest.js

```diff
@@ -214,6 +217,23 @@ export default class Suggest {
     this.autocomplete.clear();
   }
 
+  renderEmpty(query = '') {
+    if (query.trim() === '') {
+      return '';
+    }
+    return `
+      <div class="no_suggestion">
+        <div class="first_line">
+          ${_(`Impossible to find "${query}" in Qwant Maps...`)}
```

Comment:
This templated message is incorrect: the translation key should not depend on the "query".  
See https://github.com/QwantResearch/erdapfel/blob/master/src/components/OpeningHour.jsx#L40 for an example